apps/netconf/netconf_main.c

```diff
@@ -523,10 +581,24 @@ netconf_input_cb(int   s,
 		}
 		cbuf_reset(cb);
 	    }
+
+            if(chunkRemainingLength == -1) {
+                if(detect_netconf_chunk_start(buf[i], &chunkStartState, &chunkLength)) {
+                    clicon_log(LOG_INFO, "%s: detected chunk with size %i", __FUNCTION__, chunkLength);
```

Comment:
LOG_INFO syslog may be too chatty in a live system?
Can you use clicon_debug(1,..) ?